Makefile

```diff
@@ -48,56 +48,68 @@ clean: ## Remove all temporary files.
 	find -name '__pycache__' -type d -delete
 	rm -rf .coverage build
 
-install_system_dependencies: apt_get_update ## Install system-level dependencies from `debian_packages.lst`.
+install_system_dependencies: ## Install system-level dependencies from `debian_packages.lst`.
 	sudo -E apt-get install -y `tr -d '\r' < debian_packages.lst`
 
+.PHONY: static
+static: ## Collect static files for production.
+	$(HONCHO_MANAGE) collectstatic --clear --no-input
+
 manage: ## Run a management command.
 	$(HONCHO_MANAGE) $(RUN_ARGS)
 
 migrate: clean ## Run migrations.
 	$(HONCHO_MANAGE) migrate
 
-migration_check: clean ## Check for unapplied migrations.
+migrations.check: clean ## Check for unapplied migrations.
 	@!(($(HONCHO_MANAGE) showmigrations | grep '\[ \]') && printf "\n\033[0;31mERROR: Pending migrations found\033[0m\n\n")
 
-migration_autogen: clean ## Generate migrations.
+migrations: clean ## Generate migrations.
```

Comment:
Same remarks for the other changes - part of the goal of reusing Ocim's setup is to avoid the need to re-learn these commands. Making subtle changes like this would actually make it harder to remember what works on both sides..